test/dns_nsupdate/dns_nsupdate_test.rb

```diff
@@ -1,8 +1,6 @@
 require 'test_helper'
-require 'dns/dns'
-require 'dns_nsupdate/dns_nsupdate_plugin'
+require 'dns_common/dns_common'
 require 'dns_nsupdate/dns_nsupdate_main'
-require 'dns_nsupdate/dns_nsupdate_gss_main'
```

Comment:
These are needed for executing just this suite (as opposed to running all available tests). Please roll back this change